package/MDAnalysis/lib/include/calc_distances.h

```diff
@@ -50,9 +50,17 @@ inline void _minimum_image_ortho_lazy(double* x, float* box, float* half_box)
     *
     * Assumes that the maximum separation is less than 1.5 times the box length.
```

Comment:
As a thought, it would be good to have your findings somewhere (maybe like a line here to say branching is faster?). I think it'll save time next time to know someone already went ahead and looked at whether branching made this better/worse.

Reply:
In general I agree, but in this case I'm a bit hesitant because i) the function did not exist before my previous PR #3186, and ii) such things are never irrefutable truths because technology changes and such a comment will be misleading if (for whatever reason) in the future, boolean multipliers are faster than branching. Plus I didn't try CMOVs, and there's probably more things one could try out.